content/en/blog/2024/sig-node-spotlight/sig-node-spotlight.md

```diff
@@ -0,0 +1,223 @@
+---
+layout: blog
+title: "Spotlight on SIG Node"
+slug: sig-node-spotlight-2024
+date: 2024-05-09
```

Comment:
Let's push this back. How about the 21st of May?

Reply:
Perfect, will update in the next push.